README.md

```diff
@@ -60,21 +60,21 @@ This module is distributed via [npm][npm] which is bundled with [node][node] and
 should be installed as one of your project's dependencies:
 
 ```
-npm install --save prisma-extension-nested-operations
+npm install prisma-extension-nested-operations
 ```
 
 `@prisma/client` is a peer dependency of this library, so you will need to
 install it if you haven't already:
 
 ```
-npm install --save @prisma/client
+npm install @prisma/client
```

Comment:
`--save` is not needed since npm 5 (so since 2017)

Reply:
Makes sense! :)